tasks/task1.js

```diff
@@ -0,0 +1,96 @@
+// Task 1
+function summ(...args) {
```

Comment:
Add check for existence (if undefined -> return) to avoid errors

(*for every function where needed)

Reply:
add for task1-1 and 1-2:
if (args.length === 0) return null;